[1,"Error occurred during cleanup: Error: El proceso no tiene acceso al archivo porque est√° siendo utilizado por otro proceso. (os error 32): remove 'C:\\Users\\manue\\AppData\\Local\\Temp\\quarto-sessionc1b260fcf59da5fe'\n"]
